.travis.yml

```diff
@@ -2,25 +2,32 @@
 #   Generic MoveIt Travis Continuous Integration Configuration File
 #   Works with all MoveIt! repositories/branches
 #   Author: Dave Coleman, Jonathan Bohren
+dist: trusty
 language:
   - cpp
   - python
 python:
   - "2.7"
 compiler:
   - gcc
+cache:
+  - apt
```

Comment:
In fact, this `apt` is currently meaningless:
https://github.com/travis-ci/travis-ci/issues/5876#issuecomment-207661127
https://docs.travis-ci.com/user/caching/
So we don't have to add it newly.
Could you remove this line?

Reply:
Thank you for information. I fixed.
https://github.com/start-jsk/rtmros_tutorials/pull/571/commits/ada971f7912f6213b0753b725b443616a39d257d

Reply:
I confirmed, thanks